platforms/Windows/sdk.wixproj

```diff
@@ -56,6 +56,10 @@
   </Target>
 
   <ItemGroup>
+    <WixExtension Include="WixUtilExtension">
```

Comment:
This doesn’t look like it is used.

Reply:
IIRC, I needed it for the platform computation.